conf/be.conf

```diff
@@ -91,3 +91,4 @@ sys_log_level = INFO
 aws_log_level=0
 ## If you are not running in aws cloud, you can disable EC2 metadata
 AWS_EC2_METADATA_DISABLED=true
+enable_table_size_correctness_check=false
```

Comment:
why add this?

Reply:
just to skip master's core to test my code whether work normally.
after #45711 merged, I will remove this line and rebase